packages/aztecoo/src/Epetra_MsrMatrix.h

```diff
@@ -413,6 +413,6 @@ class Epetra_MsrMatrix: public Epetra_Object, public Epetra_CompObject, public v
     
 
  //! Copy constructor (not accessible to users).
-  Epetra_MsrMatrix(const Epetra_MsrMatrix & Matrix) {(void)Matrix;}
```

Comment:
The way this was originally written, I think what's happening is
```c++
Epetra_MsrMatrix(const Epetra_MsrMatrix& Matrix)
  :
  Epetra_Object(),
  Epetra_CompObject()
{
  (void)Matrix;
}
```

but it seems likely that's not what was intended.  If that was the intent, this PR should be changed to match the base class initialization above.